lib/ctcs2_to_junit.pm

```diff
@@ -122,6 +122,14 @@ sub generateXML {
                 $writer->endTag('system-err');
             }
         }
+        if ((get_var('BTRFS_PROGS')) && ($case_status eq 'failure' || $case_status eq 'skipped')) {
+            (my $test_path = $test) =~ s/-/\//;
+            $test_path = '/opt/logs/' . $test_path . '.txt';
+            my $test_out_content = script_output("if [ -f $test_path ]; then sed -n '1!H;/====== RUN /h;\${g;p;}' $test_path | head -n 100 | tr -cd '\\11\\12\\15\\40-\\46\\48-\\176'; else echo 'Test Crashed, find log in serial0.txt'; fi", 600);
```

Comment:
This is too long for a single line. Would you please make it in several lines with proper length?

Reply:
I have split this single line into 4 lines.

Reply:
well, it is not typical way ...

Anyway, merge it since small change.